ajax/libs/sticky-kit/1.1.3/sticky-kit.js

```diff
@@ -0,0 +1,270 @@
+// Generated by CoffeeScript 1.6.2
+/**
+@license Sticky-kit v1.1.3 | WTFPL | Leaf Corcoran 2015 | http://leafo.net
```

Comment:
He might forgot to update it, see: https://github.com/leafo/sticky-kit/commit/f06d8e6c3c8986672e4be11eed11feb58988c6df
That's not a dual license project, it just transfered to MIT, if I didn't remember it wrong, npm package can not be re-publish, so that may be the reason why it's old on npm, but you can still open an issue to remind author to tag the new version, and confirm ths license, at least, I'm not the author.